### PR TITLE
Add all Jensen vehicles to common.js

### PR DESCRIPTION
**Pull request by elkhorn360**
done https://github.com/Bjorn248/scca_classifier/issues/23

### CODE REVIEW

File: common.js [hunk 1]
```diff
@@ -2802,6 +2802,11 @@ const allSoloCars = {
       '2006': ['fs'],
     },
   },
+  'Jensen': {
+    'Jensen-Healey' : {
+      'all': ['cs', 'fsp', 'pd'] 
+    }
+  }
```
Comment (Bjorn248) [inline common.js:2809]: Missing a comma here which creates a syntax error.

Also please be sure to add dangling commas everywhere according to this https://eslint.org/docs/rules/comma-dangle#always-multiline

File: common.js [hunk 2]
```diff
@@ -2802,6 +2802,11 @@ const allSoloCars = {
       '2006': ['fs'],
     },
   },
+  'Jensen': {
+    'Jensen-Healey' : {
+      'all': ['cs', 'fsp', 'pd']
```
Comment (Bjorn248) [inline common.js:2807]: should be `dp` not `pd`

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
With a few tweaks this will be ready to merge

File: common.js (hunk 1 above)
Comment (elkhorn360) [reply common.js:2809]: Oh , I missed it will fix

File: common.js (hunk 2 above)
Comment (elkhorn360) [reply common.js:2807]: Oh , I missed it will fix

**Comment by Bjorn248** [comment]
Thanks!

**Pull request merged**